sdk/servicebus/Azure.Messaging.ServiceBus/tests/Processor/SessionProcessorLiveTests.cs

```diff
@@ -1202,7 +1202,9 @@ async Task ProcessMessage(ProcessSessionMessageEventArgs args)
                     {
                         var message = args.Message;
                         sessions.TryRemove(message.SessionId, out bool _);
-                        await Task.Delay(lockDuration.Add(TimeSpan.FromSeconds(10)));
+                        // wait 3x lockduration since to avoid the case where
+                        // lock is renewed at the very end of the lock duration delay
+                        await Task.Delay(lockDuration.Add(lockDuration).Add(lockDuration));
```

Comment:
You may want to consider simplifying here:

```csharp
await Task.Delay(lockDuration.Multiply(3));
```

Reply:
This isn't available in net461